aten/src/ATen/native/cpu/CatKernel.cpp

```diff
@@ -57,19 +54,19 @@ void cat_serial_kernel_impl(Tensor& result, TensorList tensors, int64_t dim) {
             input_ptr,
             local_inner);
       }
-      offset += local_inner;
+      result_ptr += local_inner;
     }
   }
 }
 
-void cat_serial_kernel(Tensor& result, TensorList tensors, int64_t dim) {
-  AT_DISPATCH_FLOATING_TYPES(result.scalar_type(), "cat_serial_kernel", [&]() {
-    cat_serial_kernel_impl<scalar_t>(result, tensors, dim);
+void cat_contig_kernel(Tensor& result, TensorList tensors, int64_t dim) {
```

Comment:
The renaming is a bit confusing. Previously, we had: cat_serial_stub -> cat_serial_kernel -> cat_serial_kernel_impl. Everything followed the same naming pattern.

Now it is: cat_serial_stub -> cat_contig_kernel -> cat_contig_kernel_impl.

Renaming is fine if the kernel is no long single-threaded, but it looks like it still is single-threaded. Additionally, if you are going to rename the kernel, please also rename `cat_serial_stub` to match and https://github.com/pytorch/pytorch/blob/c6d301220a89908df0f346db5bf391218f8dd068/aten/src/ATen/native/cpu/CatKernel.h#L8

Reply:
My bad again. I meant to keep the name same as before, but the PR went through a series of changes and in that I half reverted my original changes. I will fix it.